functions/src/on-create-loi.ts

```diff
@@ -51,6 +51,13 @@ export async function onCreateLoiHandler(
       properties,
       wkt
     );
+
+    Object.keys(properties).forEach(key => {
```

Comment:
You could also use `filter()` to only modify the object properties and leave the rest along.

Reply:
e.g. 
```ts
Object.keys(properties)
  .filter(key => typeof properties[key] === 'object')
  .forEach(key => { properties[key] = JSON.stringify(properties[key]); }
```

Does this work?